config/urls.py

```diff
@@ -41,6 +44,15 @@
     path("graphql/", CustomAsyncGraphQLView.as_view(schema=schema, graphiql=False)),
     path('api/', include(router.urls)),
     path('tinymce/', include('tinymce.urls')),
+
+    path('admin/login/', auth_views.LoginView.as_view(template_name='login.html'), name='login'),
+    path("password_reset", views.password_reset_request, name="password_reset"),
+    path('password_reset/done/', auth_views.PasswordResetDoneView.as_view(
+        template_name='password_reset_done.html'), name='password_reset_done'),
+    path('reset/<uidb64>/<token>/', auth_views.PasswordResetConfirmView.as_view(
+        template_name="password_reset_confirm.html"), name='password_reset_confirm'),
+    path('reset/done/', auth_views.PasswordResetCompleteView.as_view(
+        template_name='password_reset_complete.html'), name='password_reset_complete'),
```

Comment:
This should be enough
```suggestion
    path(
        r'^password/reset/<uidb64>/<token>/$',
        auth_views.PasswordResetConfirmView.as_view(
            success_url=reverse_lazy('login'),
            ),
        name='password_reset_confirm'
    ),
```